### DISCUSSION TITLE
s.entries is not a function

### DISCUSSION THREAD

**Discussion by Bilo2110** [ulic75/power-flow-card#176] [Q&A] [answered]
Hi after update 0.1.8 if i want to modify the card i see this error:

visual editor is not supported for this configuration:
   s.entries is not a function
you can modify your configuration in YAML

type: custom:power-flow-card-plus
entities:
  battery:
    entity:
      '0': s
      '1': e
      '2': 'n'
      '3': s
      '4': o
      '5': r
      '6': .
      '7': z
      '8': c
      '9': s
      '10': _
      '11': b
      '12': a
      '13': t
      '14': t
      '15': e
      '16': r
      '17': 'y'
      '18': _
      '19': p
      '20': o
      '21': w
      '22': e
      '23': r
      production: sensor.inverter_potenza_carica
      consumption: sensor.inverter_potenza_scarica
    state_of_charge: sensor.solarm_battery_soc
  grid:
    entity:
      '0': s
      '1': e
      '2': 'n'
      '3': s
      '4': o
      '5': r
      '6': .
      '7': z
      '8': c
      '9': s
      '10': _
      '11': t
      '12': o
      '13': t
      '14': a
      '15': l
      '16': _
      '17': g
      '18': r
      '19': i
      '20': d
      '21': _
      '22': p
      '23': o
      '24': w
      '25': e
      '26': r
      production: sensor.inverter_potenza_esportata
      consumption: sensor.inverter_potenza_importata
    invert_state: true
  solar:
    entity: sensor.potenza_totale_istantanea
    display_zero_state: true
  home:
    entity: sensor.consumo_istantaneo_channel_1_power
    override_state: true
    name: Casa
    icon: mdi:home
clickable_entities: true
display_zero_lines:
  mode: show
  transparency: 50
  grey_color:
    - 189
    - 189
    - 189
use_new_flow_rate_model: true
w_decimals: 0
kw_decimals: 1
min_flow_rate: 0.75
max_flow_rate: 6
min_expected_power: 0.01
watt_threshold: 1000
transparency_zero_lines: 0
max_expected_power: 3000

**Comment by ulic75** [comment, marked as answer]
This is the power-flow-card not the power-flow-card-plus repository.
